include/iotc.h

```diff
@@ -276,50 +250,17 @@ extern iotc_state_t iotc_connect_to(iotc_context_handle_t iotc_h,
                                     iotc_user_callback_t* client_callback);
 
 /**
- * @brief Publish a message to Cloud IoT Core on a given topic.
- *
- * To publish a message, the device must already be connected to Cloud IoT Core.
- * Please see <code>iotc_connect()</code> or <code>iot_connect_to()</code> to
- * connect to Cloud IoT Core.
- *
- * The callback function has the following signature:
- *  <pre>
- *  void foo( iotc_context_handle_t in_context_handle
- *                , void* user_data
- *                , iotc_state_t state )
- *  </pre>
- *
- * The callback parameters are:
- *   - in_context_handle: A context handle provided to
- * <code>iotc_publish()</code>.
- *   - user_data: (Optional) The value that the client application passed to the
- * <code>iotc_publish()</code>.
- *   - state: IOTC_STATE_OK The publication has succeeded. For information on
- * other error codes, see the <a href="../../../porting_guide.md">porting
- * guide</a> or <code>examples/</code> directory.
- *
- *
- * @param [in] iotc_h A context handle created by invoking iotc_create_context.
- * @param [in] topic A string based topic name that you have created for
- *     messaging via the IoT Core webservice.
+ * @brief Publishes a message to an MQTT topic.
+ *
+ * @param [in] iotc_h A {@link iotc_create_context() context handle}.
+ * @param [in] topic The MQTT topic.
  * @param [in] msg The payload of the message.
  * @param [in] qos The Quality of Service (QoS) level. Can be <code>0</code>,
- * <code>1</code>, or <code>2</code>. Cloud IoT Core doesn't support QoS
- * level 2. For more information, see the MQTT specification or
- * <code>iotc_mqtt_qos_e()</code> in <code>iotc_mqtt_message.h</code>.
+ *     <code>1</code>, or <code>2</code>.
```

Comment:
2 isn't supported neither by Cloud IoT Core MQTT bridge nor by the Client SDK. So please remove the 2.

Reply:
I think we should state explicitly that QoS 2 isn't supported. Since, actually, it is rarely supported by other vendors either.

Reply:
Done